src/auction/README.md

```diff
@@ -0,0 +1,53 @@
+# Auction
+
+## Install
+
+```bash
+$ npm install @magnite/fledge.polyfill
+```
+
+## Usage
+
+```html
+<script type="module">
+    import { fledge } from "./node_modules/@magnite/fledge.polyfill/esm/index.js";
+
+    const options = {
+        "seller": "www.seller.com",
+        "decision_logic_url": "https://ssp.com/auction",
+        "interest_group_buyers": [
+            "www.buyer1.com",
+            "www.buyer2.com",
+        ],
+    };
+
+    const auctionResults = await fledge.runAdAuction(options);
+</script>
+```
+
+## API
+
+### runAdAuction(options)
+
+Returns `true` and creates an entry in a cross-domain Indexed dB store.
```

Comment:
looks like you are now returning a token (which is good) should update this to be "returns promise which resolves with a token" or similar.